sdks/java/core/src/main/java/org/apache/beam/sdk/util/MergingActiveWindowSet.java

```diff
@@ -144,7 +144,7 @@ public void ensureWindowIsActive(W window) {
     checkState(stateAddressWindows != null,
                              "Cannot ensure window %s is active since it is neither ACTIVE nor NEW",
                              window);
-    if (stateAddressWindows.isEmpty()) {
+    if (stateAddressWindows != null && stateAddressWindows.isEmpty()) {
```

Comment:
It's a findBugs error after compiling with edt.   Basically, findbugs says that since there is a possibility of stateAddressWindows being null (since you DO check for it above), then there is a possibility of isEmpty being called on a null reference.